File: Nian_Karl/QL_Nian_Karl/src/org/uva/sea/ql/AST/value/DateValue.java
package org.uva.sea.ql.AST.value;

import java.util.Date;

public class DateValue extends AbstractValue<Date> {

	private Date date;

	public DateValue(Date date) {
		this.date = date;
	}

	
	@Override
	public Date getValue() {
		return date;
	}

}

File: CarlaErik/UvA.SoftCon.Questionnaire/Runtime/Evaluation/Types/DateValue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using UvA.SoftCon.Questionnaire.QL.AST.Model.Statements;

namespace UvA.SoftCon.Questionnaire.Runtime.Evaluation.Types
{
    public class DateValue : Value<DateTime>
    {
        public override DataType DataType
        {
            get
            {
                return DataType.Date;
            }
        }

        public DateValue(DateTime value)
            : base(value) { }

    }
}
